doc/spinel-protocol-src/spinel-prop-phy.md

```diff
@@ -46,7 +46,7 @@ that is supported by the underlying radio hardware.
 
 ### PROP 37: PROP_PHY_TX_POWER {#prop-phy-tx-power}
 * Type: Read-Write
-* Packed-Encoding: `c` (int8)
+* Packed-Encoding: `L` (uint32)
```

Comment:
This breaks compatibility with older wpantund.

How about if we keep the `PHY_TX_POWER` as a `int8`. Inside `NcpBase` we simply convert the value to `uint32_t`. So the old model still sort of works.

And then add a new spinel property `PHY_TX_POWER_CONFIG` with `uint32` format which is tied to the new APIs. 

Thoughts?

Reply:
I was waiting for you to call this out :)

One challenge: is the NCP expected to convert between `PHY_TX_POWER` 
 and `PHY_TX_POWER_CONFIG` values?  For example, do we require all NCPs that support `PHY_TX_POWER_CONFIG` to also provide an appropriate mapping for `PHY_TX_POWER`?

Reply:
I think it's definitely not the job of OT (or the NCP) code to convert.
How about delegating this to platform radio driver?
- Radio required to provide a function to convert tx power in dBm to tx config (`uint32`) and also the reverse, OR
- Radio provides function to get/set tx power as dBm in addition to new functions for setting/getting it as a tx power config? (basically requiring both models)...

Reply:
The mapping from dBm to power config may not be 1-to-1.

Reply:
I understand that but in that case the radio driver can try to do best conversion that makes sense. We can also consider adding a specific value to indicate "invalid/bad" tx power.  

This is just a solution to keep the `PHY_TX_POWER` property (and have it be somewhat compatible with older code). We can over time deprecate this property.